src/core/js/startController.js

```diff
@@ -21,7 +21,27 @@ define([
             var hasStartId = (startId)
                 ? true
                 : false;
+            
+            if (hasStartId) {
```

Comment:
This section won't work,  as explained on [this comment](https://github.com/adaptlearning/adapt_framework/issues/2392#issuecomment-483215043). Adding classes to the page through the `startController` won't keep them permanently on the page.

Reply:
I agree that this section will execute only once  in a lifetime of application. But I would put this in a different way that if we are saying  `"_skipIfComplete": true` then it is desirable that we should not apply the class and we should allow navigation buttons to show. 

OR
 
The `"_skipIfComplete": true`  is only to avoid jumping to the specified page and rest of the things should work as is i.e. class should be applied even if we are saying "skip"?
I think this is contradictory behaviour.

Reply:
> It would be incredibly useful [...] to [be able to] add a custom class to the html tag.

> Preference is for the class to be [...] defined so appropriate navigational elements, such as back or home buttons, can be removed in the base theme

> ... The class would be removed when the user navigates away from the [...] page.

This behaviour is made complicated by the original description. It does not need to be dependent on `"_skipIfComplete": true` or the `startController`. We can achieve the same outcome by allowing `_htmlClassName` only.

Reply:
i think, we will remove the start controller logic then and will add class directly.